xquery/str.xqm

```diff
@@ -147,7 +147,9 @@ declare function str:txtFromTEI($nodes as node()*, $lang as xs:string) as xs:str
         	if($node/@cert) then ($node/child::node() ! str:txtFromTEI(., $lang), '(?)') 
         	else $node/child::node() ! str:txtFromTEI(., $lang)
         case element(tei:del) return ()
+        case element(tei:orig) return ()

```

Comment:
Please keep it, this is needed to remove the `<orig>` from the output

Reply:
I agree! I added this line, because in my usecase `<orig>` appeared ;-)